sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobOutputStream.java

```diff
@@ -249,7 +249,7 @@ private synchronized void commit() throws StorageException {
             return Mono.empty();
         }
 
-        if (this.streamType == BlobType.PAGE_BLOB && (writeLength % Constants.PAGE_SIZE != 0)) {
+        if (this.streamType == BlobType.PAGE_BLOB && (writeLength % PageBlobAsyncClient.PAGE_BYTES != 0)) {
```

Comment:
Some of these constants that are on the client types were intentionally put on the clients and made public so they could be used by customers. I think thvalues like these should be kept on the client, perhaps?

Reply:
I moved this over from the `Constants` based value to the client specified one as I thought it would keep inline with the PageBlob page size value. Both were 512 and I thought it was strange to keep track of this in two separate places.

Reply:
You're right. I thought this was going the other direction!